### PR TITLE
Remove selfplay override for root has own cpuct params

### PR DESCRIPTION
**Pull request by Tilps**
no longer needed as its always sent by training server.

### CODE REVIEW

File: src/selfplay/tournament.cc [hunk 1]
```diff
@@ -127,7 +127,6 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
                              "multiplexing");
   defaults->Set<bool>(SearchParams::kStickyEndgamesId.GetId(), false);
   defaults->Set<bool>(SearchParams::kLogitQId.GetId(), false);
-  defaults->Set<bool>(SearchParams::kRootHasOwnCpuctParamsId.GetId(), false);
```
Comment (mooskagh) [inline src/selfplay/tournament.cc]: Also remove the parameter from `params.cc`? As the only reason it existed is actually setting it to false for selfplay.

Comment (Tilps) [reply src/selfplay/tournament.cc]: We can't remove it - the server still has to send it until the server enforces a binary where the default is true - Which is what I am doing here.

Comment (borg323) [reply src/selfplay/tournament.cc]: Add a comment so we don't forget?

Comment (Tilps) [reply src/selfplay/tournament.cc]: over in params.cc - sure.

**Pull request merged**